api.php

```diff
@@ -351,6 +351,48 @@ public static function delete_pdf( $form_id, $pdf_id ) {
 		return $options->delete_pdf( $form_id, $pdf_id );
 	}
 
+	/**
+	 * Retrieve the filename of the PDF document
+	 * The .pdf extension is not returned by default (pass true to the 3rd param)
+	 *
+	 * See https://docs.gravitypdf.com/v6/developers/api/get_pdf_filename/ for more information about this method
+	 *
+	 * @param int    $entry_id          The Gravity Form entry ID
+	 * @param string $pdf_id            The Gravity PDF ID number (the pid number in the URL when viewing a setting in the admin area)
+	 * @param bool   $include_extension Whether to include the .pdf extension in the filename
+	 *
+	 * @return string|WP_Error   Return the filename of the PDF, or a WP_Error on failure
+	 *
+	 * @return string|WP_Error
+	 *
+	 * @since 6.12
+	 */
+	public static function get_pdf_filename( $entry_id, $pdf_id, bool $include_extension ) {
```

Comment:
@jakejackson1 move to another PR. Not required here.

Will also add an API for getting the field markup for a PDF.